adafruit_ht16k33/segments.py

```diff
@@ -251,17 +252,16 @@ def _text(self, text: str) -> None:
         for character in text:
             self._push(character)
 
-    def _number(self, number: Union[int, float], decimal: int = 0) -> str:
+    def _number(self, number: float, decimal: int = 0) -> str:
```

Comment:
This should be able to handle floats fine.

Reply:
Same thing, `int` is a subset of `float`